### PR TITLE
Makes synthflesh patches heal more 

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Makes synthflesh heal 1 brute and 1 burn per cycle. This mostly affects patches but will cause automenders to heal slightly more

## Why It's Good For The Game
Styptic powder and silver sulfadiazene are great in patches because they heal 2 of their respective damagetype while in the system, even with the small amounts that patches inject
Synthflesh didn't have this so it was heavily outgunned in terms of healing in patchform, making the synthflesh patches that ERT and other Centcomm jobs get a pure meme

## Images of changes
This was after dealing 75 brute and 75 burn damage to myself
image

## Testing
See above

## Changelog
:cl:
tweak: Synthflesh now heals 1 brute and 1 burn for each cycle that it is inside of you. Mostly affects patches which will now heal a lot more
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 1]
```diff
@@ -296,6 +296,12 @@
 	taste_description = "blood"
 	goal_difficulty = REAGENT_GOAL_NORMAL
 
+/datum/reagent/medicine/heal_on_apply/synthflesh/on_mob_life(mob/living/M)
+	var/update_flags = STATUS_UPDATE_NONE
+	update_flags |= M.adjustBruteLoss(-1 * REAGENTS_EFFECT_MULTIPLIER, FALSE)
+	update_flags |= M.adjustFireLoss(-1 * REAGENTS_EFFECT_MULTIPLIER, FALSE)
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/reagents/medicine.dm:302]:
```suggestion
	update_flags |= M.adjustBruteLoss(-1.25 * REAGENTS_EFFECT_MULTIPLIER, FALSE)
	update_flags |= M.adjustFireLoss(-1.25 * REAGENTS_EFFECT_MULTIPLIER, FALSE)
```

**Review by DGamerL — COMMENTED** [review]
_As Qwerty:_ I think this is better

File: code/modules/reagents/chemistry/reagents/medicine.dm (hunk 1 above)
Comment (DGamerL) [reply code/modules/reagents/chemistry/reagents/medicine.dm:302]: Aye aye Capn'

**Comment by CinnamonSnowball** [comment]
There's already #25574 that completely replaces ERT's Synthflesh patches with something useful, if that's the reason to buff Synthflesh here.

This change, as the stats currently are, will make Synthflesh patches completely outperform Styptic Powder/Silver Sulfadiazine patches in most cases - the only time you'd want the classic patches would be if for some reason you plan on using the extra damage healed on one of the damage types, as even small Synthflesh patches would heal enough damage to take care of most injuries you would still consider using only a singular medicine on:
​ ​ 10u Styptic Powder/Silver Sulfadiazine patch = ~123 brute/burn total
​ ​ 10u Styptic Powder + Silver Sulfadiazine patch together = ~48 brute + burn total accounting for shorter duration
​ ​ 10u Synthflesh patch = ~86 brute + burn total

As a medical main, this saddens me, as I'd much rather have more *specialized* options for medicine that can interact with each other, than fewer simple fix-all chemicals that trump multiple other medicines with minimal downsides. As an on-topic example, currently applying more than one patch greatly shortens their duration and thus limits their healing, encouraging you to use a patch on the greater damage type while treating the lesser one with other medication.

I also feel like this change is sort of.. unnecessary? It detracts from Synthflesh being unique in being a powerful healing medicine with the drawback of having a fairly strict application method, while buffing the *improper* use of Synthflesh. If the reason for this change is only that ERTs happen to have Synthflesh patches, I feel like the proper fix is to simply update their loadout to no longer use gravely outdated medicine, as the PR I mentioned above does.

**Comment by DGamerL** [comment]
> There's already https://github.com/ParadiseSS13/Paradise/pull/25574 that completely replaces ERT's Synthflesh patches with something useful, if that's the reason to buff Synthflesh here.

It's not, I'm just dissatisfied with Synthflesh's inability to work normally in patches

> This change, as the stats currently are, will make Synthflesh patches completely outperform Styptic Powder/Silver Sulfadiazine patches in most cases 

This is assuming the damage is evenly spread. In most cases you'll have a lot more of one damagetype, for which the respective patches will work better. The intention is that synthflesh patches are an easy general heal, being a bit slower than the specialized chems but healing more overall

**Review by Sirryan2002 — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**